rmgpy/rmg/react.py

```diff
@@ -104,6 +104,112 @@ def reactMolecules(moleculeTuples):
     return reactionList
 
 def deflate(rxns, molecules, reactantIndices):
+def findDegeneracies(rxnList, useSpeciesReaction = True):
+    """
+    given a list of reaction object with Molecule objects, this method 
+    removes degenerate reactions and
+    increments the degeneracy of the reaction object. This method modifies
+    rxnList in place and does not return anything.
+
+    This algorithm used to exist in family.__generateReactions, but was moved
+    here because it didn't have any family dependence.
+    """
+    # These duplicates should be combined and the reaction degeneracy should be increased
+    # Reaction degeneracy is only increased if two reaction are isomorphic but resulted
+    # from different transition states.
```

Comment:
So if I came across this comment randomly in the code without reading the function I would have thought the above comments implied that multiple TS's were simply treated as degeneracies for purposes of this function especially considering the docstring doesn't mention multiple TS's being added as duplicate reactions.  Can you make that a bit more clear?